src/pgduckdb_options.cpp

```diff
@@ -115,36 +116,25 @@ ReadDuckdbExtensions() {
 }
 
 static bool
-DuckdbInstallExtension(Datum name) {
+DuckdbInstallExtension(Datum name_datum) {
 
-	auto extension_name = DatumToString(name);
+	auto extension_name = DatumToString(name_datum);
 	auto install_extension_command = duckdb::StringUtil::Format("INSTALL %s;", extension_name.c_str());
-	{
-		auto connection = pgduckdb::DuckDBManager::CreateConnection();
-		auto res = connection->context->Query(install_extension_command, false);
-
-		if (res->HasError()) {
-			elog(WARNING, "(PGDuckDB/DuckdbInstallExtension) %s", res->GetError().c_str());
-			return false;
-		}
-	}
-
-	bool nulls[Natts_duckdb_extension] = {0};
-	Datum values[Natts_duckdb_extension] = {0};
-
-	values[Anum_duckdb_extension_name - 1] = name;
-	values[Anum_duckdb_extension_enable - 1] = 1;
-
-	/* create heap tuple and insert into catalog table */
-	Relation duckdb_extensions_relation = relation_open(ExtensionsTableRelationId(), RowExclusiveLock);
```

Comment:
Is `ExtensionsTableRelationId` still used somewhere now?

Reply:
In `ReadDuckdbExtensions`